src/c++/perf_analyzer/client_backend/client_backend.h

```diff
@@ -171,7 +241,7 @@ struct InferOptions {
   /// The unique identifier for the sequence being represented by the
   /// object. Default value is 0 which means that the request does not
   /// belong to a sequence.
-  uint64_t sequence_id_;
+  SequenceId sequence_id_;
```

Comment:
How would perf_analyzer know whether to use the integer sequence id or string sequence id?